### PR TITLE
LPS-154857 rename properties: module.framework.properties.dependency.* , module.framework.properties.initial.system.check.enabled

### PR DESCRIPTION
**Pull request by W2jx**
Hi @dantewang, I fixed the problem in the code.

### CODE REVIEW

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@W2jx Code conflicts, please rebase.

Tina.

**Comment by tinatian** [comment]
@zhangyan0701 Please help me scan test results.

Tina.

File: portal-impl/src/com/liferay/portal/util/PropsValues.java [hunk 1]
```diff
@@ -982,6 +982,10 @@ public class PropsValues {
 			PropsUtil.get(
 				PropsKeys.INDEX_SEARCH_QUERY_SUGGESTION_SCORES_THRESHOLD));
 
+	public static final boolean INITIAL_SYSTEM_CHECK_ENABLED =
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PropsValues.java:985]: @W2jx Who is using this constant?

Tina

**Comment by tinatian** [comment]
@W2jx For the constants you added to PropsValues, please remove it if no one is using it. We only add it when it is used.


Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**